### PR TITLE
Load test data in sda

### PR DESCRIPTION
**Pull request by MalinAhlberg**
This PR adresses part 1 of #4 .

When starting the gdi containers, data from #17 is loaded to the sensitive data archive. Instructions for downloading it can be found [here](https://github.com/NBISweden/ejprd/blob/feat/use_test_data/docs/htsget.md) and instructions on how to access it via htsget can be found [here](https://github.com/NBISweden/ejprd/blob/feat/use_test_data/docs/storage-and-interfaces.md#download-unencrypted-files-directly).

For info of the binary `sda-cli` see [this repo](https://github.com/NBISweden/sda-cli) ([version used](https://github.com/NBISweden/sda-cli/releases/tag/v0.1.0)).

### CODE REVIEW

File: config/gdi-starter-kit/docker-compose.override.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ services:
     healthcheck:
       test: curl -fq http://localhost:9000/minio/health/live
     restart: no
-    ports: !override []
+    # ports: !override []
```
Comment (wna-se) [inline config/gdi-starter-kit/docker-compose.override.yml:28]: I would keep this or remap the ports to expose them here instead, ideally with an environment variable to be able to configure the external port. E.g., 
```
    ports: !override
      - ${PORT:-1234}:1234
```

**Review by wna-se — COMMENTED** [review]
Works well! Might be useful to also load the BAM file used in the containerised compute example.

File: config/gdi-starter-kit/docker-compose.override.yml (hunk 1 above)
Comment (MalinAhlberg) [reply config/gdi-starter-kit/docker-compose.override.yml]: yes, sorry, this was just intended for testing, but happened to be commited. Reset in 01bd15d

**Comment by MalinAhlberg** [comment]
> Might be useful to also load the BAM file used in the containerised compute example.

Sure. Or do we have any bam file more related to this project?

**Pull request merged**